tests/test_trade.py

```diff
@@ -78,6 +87,54 @@ def test_init_only(self):
         """Tests base LP setups"""
         self.run_base_trade_test(agent_policies=[], target_liquidity=1e6, target_pool_apr=0.05)
 
+    def test_compare_agent_to_calc_liquidity(self):
+        """Compare agent init as above to old calc_liquidity method"""
+        for target_liquidity in (1e2, 1e3, 1e4, 1e5, 1e6, 1e7, 1e8, 1e9):
+            for target_pool_apr in (0.01, 0.03, 0.05, 0.10, 0.25, 0.5, 1, 2):  # breaks at 500%
+                print(f"running test with {target_liquidity=} and {target_pool_apr=}")
+                simulator = self.run_base_trade_test(
+                    agent_policies=[],
+                    target_liquidity=target_liquidity,
+                    target_pool_apr=target_pool_apr,
+                    init_only=True,
+                )
+                # assign the results of the init_lp agent to explicit variables
+                share_reserves_new = simulator.market.market_state.share_reserves
+                share_reserves_old, bond_reserves_old = simulator.market.pricing_model.calc_liquidity(
+                    market_state=simulator.market.market_state,  # used only for share_price and init_share_price
+                    target_liquidity=target_liquidity,
+                    target_apr=target_pool_apr,
+                    position_duration=simulator.market.position_duration,
+                )
+                market_old = Market(
```

Comment:
nit: `market_old` is only meaningful for us because we recall how this used to be done. I think a more descriptive variable name would be helpful here. e.g. `market_no_lp` vs `market_with_lp`